move_cart_to_ship:
  ros__parameters:
    init_position: [0.0272, 0.1311, -0.0099, 1.0000]
    loading_position: [4.4079, -0.1370, -0.7175, 0.6965]
    face_shipping_position: [2.0453, -0.0101, 0.7267, 0.6869]
    shipping_position: [2.0282, 1.0123, 0.6864, 0.7272]

